cmd/ftl-cron/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/TBD54566975/ftl"
+	"github.com/TBD54566975/ftl/backend/cron"
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/ftlv1connect"
+	_ "github.com/TBD54566975/ftl/internal/automaxprocs" // Set GOMAXPROCS to match Linux container CPU quota.
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/observability"
+	"github.com/TBD54566975/ftl/internal/rpc"
+)
+
+var cli struct {
+	Version             kong.VersionFlag     `help:"Show version."`
+	ObservabilityConfig observability.Config `embed:"" prefix:"o11y-"`
+	LogConfig           log.Config           `embed:"" prefix:"log-"`
+	CronConfig          cron.Config          `embed:""`
```

Comment:
In general we should prefix all embedded configs.

Reply:
That is not the pattern we have been using for the 'main' config though. Cron, Runner, Provisioner, Controller etc all have one 'main' config that is not prefixed.